src/go/rpk/pkg/cli/cmd/cluster/config/import.go

```diff
@@ -84,6 +84,12 @@ func importConfig(
 			// For types that aren't numeric or array, pass them through as-is
 		}
 
+		// We are excluding cluster_id since is a variable stored as a config
```

Comment:
`We exclude cluster_id from upsert here and remove below to avoid any accidental duplication of the ID from one cluster to another`.

However, I don't think the cluster ID should be user chosen nor should it be modifiable. cc @jcsp